src/learning-header/LearningHeader.jsx

```diff
@@ -52,7 +51,7 @@ function LearningHeader({
         </div>
         {showUserDropdown && authenticatedUser && (
           <AuthenticatedUserDropdown
-            username={authenticatedUser.username}
+            name={getConfig().SHOW_FULLNAME == 'true' ? authenticatedUser.name : authenticatedUser.username}
```

Comment:
A bit of a personal opinion but rather then checking for a string value we could just consider SHOW_FULLNAME to be true if it is defined irrespective of the value  soo
`
name={getConfig().SHOW_FULLNAME ? authenticatedUser.name : authenticatedUser.username}
`

But his works great too